python/TopPairElectronPlusJetsSelectionFilter_cfi.py

```diff
@@ -9,7 +9,7 @@
     VertexInputTag = cms.InputTag('offlineSlimmedPrimaryVertices'),
 
     # Lepton cuts
-    minSignalElectronPt=cms.double(30.),
+    minSignalElectronPt=cms.double(34.),
```

Comment:
BTW: Similar argument to lower jet pT: IF we want to evaluate trigger and/or lepton energy scale, the selection should allow for the necessary wiggle-room